Modules/Clients/Tests/Feature/CustomersTest.php

```
@@ -415,6 +415,44 @@ public function it_fails_to_delete_customer_when_contact_attached(): void
+        $this->assertNotNull(Relation::find($customerA->id));               // A is visible to tenant A
+        $this->assertNull(Relation::find($customerB->id));                  // B is NOT visible to tenant A
+    }
+
+    #[Test]
+    #[Group('multi-tenancy')]
+    public function it_only_lists_customers_for_the_current_tenant(): void
+    {
+        /* Arrange */
+        $companyB = \Modules\Core\Models\Company::factory()->create();
+
+        $customerA = Relation::factory()->for($this->company)->customer()->create(['company_name' => 'Visible Customer']);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$customerA'. (undefined)

(UnusedLocalVariable)